packages/backend/src/tests/generated/db.ts

```diff
@@ -36,7 +36,7 @@ export type PgCatalogInt2 = number;
 /**
  * A PG numeric
  */
-export type PgCatalogNumeric = number;
+export type PgCatalogNumeric = string;
```

Comment:
Can you add a comment to the JSON schema types explaining why this is a string so users understand what's going on (i.e. precision)